src/main/content/_layouts/javadocs.html

```diff
@@ -9,6 +9,11 @@
 
     {% include doc_header.html %}
 
+    <div id="header_background" class="black_blue_gradient_background">
+        <div id="docs_container" class="container">
```

Comment:
We may be able to delete `#docs_container` and move the dark background into `doc_header.html`.  My thinking is that the dark background is now only used for the header, where as before we needed the dark background to stretch all the way down the entire page.